data/members/cryptas.yaml

```
@@ -38,7 +38,7 @@ social:
   twitter: 
   facebook: 
   instagram: 
-  linkedin: https://www.linkedin.com/company/cryptas
+  linkedin: https://www.linkedin.com/cryptas
```

This page can't be found but the original URL seems to be fine.
```suggestion
  linkedin: https://www.linkedin.com/company/cryptas
```